terraform/aws-accounts/cloud-platform-aws/vpc/eks/cluster.tf

```diff
@@ -17,6 +17,25 @@ provider "kubernetes" {
   load_config_file       = false
 }
 
+locals {
```

Comment:
this is already in, see main.tf from https://github.com/ministryofjustice/cloud-platform-infrastructure/pull/1201